File: gatling-performance-suite/src/test/scala/com/example/performance/scenarios/UserJourneyScenario.scala
package com.example.performance.scenarios

import io.gatling.core.Predef._
import scala.concurrent.duration._

object UserJourney {

  import HomepageScenario._
  import SearchScenario._

  val scenarioDef = scenario("User Journey")
    .exec(homepage)
    .pause(1)
    .exec(search)
}

File: src/test/java/com/example/performance/scenarios/UserJourneyScenario.java
package com.example.performance.scenarios;

import io.gatling.javaapi.core.ScenarioBuilder;

import static io.gatling.javaapi.core.CoreDsl.*;

public class UserJourneyScenario {

    public static ScenarioBuilder scenarioDef = scenario("User Journey")
            .exec(HomepageScenario.homepage)
            .pause(1)
            .exec(SearchScenario.search);
}